docs/design/storagenode-downtime-tracking.md

```diff
@@ -66,28 +64,28 @@ ORDER BY
 LIMIT 1
 ```
 
-The process runs the query repeatable until it doesn't return records then it ends and will run again in the next interval. The query only gets one record at the time for not holding a record set during the process.
+The chore repeats the query until all nodes have been handled. It avoids selecting too many nodes at the same time.
```

Comment:
The problem to me isn't selecting many nodes at the same time, it's more about to hold the result set.

I don't know if that's clear for everybody, why the limit is 1, for that reason, was mentioned.

Reply:
Why would holding the record set be problematic?

Reply:
I may be wrong, but I thought that holding the record open too long isn't good for the DB.

If it's fine, then we don't need to run this query one per one.

Reply:
Clarified in slack, there is no problem for the DB if we don't use a cursor.

Let's merge this and I will remove the limit in the parent branch and rewrite what references to it.